source/configuration_manual/howto/chasquid_and_dovecot_sasl.rst

```diff
@@ -0,0 +1,65 @@
+.. _howto-chasquid_and_dovecot_sasl:
+
+=========================
+chasquid and Dovecot SASL
+=========================
+
+`chasquid <https://blitiri.com.ar/p/chasquid>`__ users can use :ref:`Dovecot
+SASL <sasl>` authenticating SMTP clients.
+
+This is supported from version 0.04 and later, and uses the *PLAIN*
+mechanism (only).
+
+conf.d/10-master.conf
+---------------------
+
+The following needs to be added to the Dovecot configuration, usually in
+*/etc/dovecot/conf.d/10-master.conf*:
+
+::
+
+   service auth {
+     unix_listener auth-chasquid-userdb {
+       mode = 0660
+       user = chasquid
+     }
+     unix_listener auth-chasquid-client {
+       mode = 0660
+       user = chasquid
+     }
+   }
+
+If chasquid is running under a different user, adjust the *user =* lines
+accordingly.
+
+chasquid.conf
+-------------
+
+Add the following line to */etc/chasquid/chasquid.conf*:
+
+::
+
+   dovecot_auth: true
+
+That should be it, because chasquid will "autodetect" the full path to
+the dovecot sockets, by looking in the usual places (tested in Debian,
+Ubuntu, and CentOS).
+
+If chasquid can't find them, the paths can be set with the
+*dovecot_userdb_path* and *dovecot_client_path* options (see the
+configuration manual page for details).
```

Comment:
this could be a link?

Reply:
I searched the chasquid website and their configuration docs but there is not really a linkable place for these settings. There is only one mention of these configs in the manpages of chasquid.